src/js/popup/actions/timer.js

```diff
@@ -50,15 +50,15 @@ export function isTimerActive(data) {
   return data.timer.startTime > 0;
 }
 
-export function onTimerClick(store, selectedProjectId, selectedMergeRequestId) {
-  const data = store.getState();
-  if (!isTimerLoaded(data)) return;
+export function onTimerToggled(store) {
```

Comment:
Can you please add TODO to make it proper redux-thunk action. Just to not forget.